src/utils/clientToSVGTranslator.ts

```diff
@@ -70,6 +70,9 @@ export module Utils {
 
       return scaledPosition;
     }
+    public isWithinTarget(e: MouseEvent) {
```

Comment:
Probably should be renamed to "`insideSVG()`", and take a generic `Event`. (Renaming it also makes the method self-documenting).